tt_metal/jit_build/CMakeLists.txt

```
@@ -8,3 +8,18 @@ set(JIT_BUILD_SRCS
 
 add_library(jit_build OBJECT ${JIT_BUILD_SRCS})
 target_link_libraries(jit_build PUBLIC common)
+
+find_program(GIT_EXECUTABLE gi)
```

Did this work as-is?  🤔 

```suggestion
find_program(GIT_EXECUTABLE git)
```

---

My bad, I was testing what happens if 'git' is not present in the system. Ended up committing that change.
It turns out, find_program will find 'git', even if I ask it find 'gi'.
I feel the 'result_variable' is more reliable to catch any errors, find_program doesn't catch all issues. If a system doesn't have 'git', but has some other application whose name starts with 'git', it might return that application.
@blozano-tt

---

All `find*` will store positive results in the CMakeCache.txt for efficiency the next time.  Unfound things will be re-evaluated in case they've shown up since the last run.

That means to test a found -> not found, you'll need to remove the entry from CMakeCache.txt (or just blow away the entire file).